application/views/themeOptions/options_core.php

```diff
@@ -126,7 +126,7 @@
                                         $sParentLabelOption = $options[$sParentOption];
                                         $options['inherit'] = gT("Inherit") . ' [' . gT($sParentLabelOption) . ']';
                                     } else {
-                                        $sParentOption = $sParentOption !== '' ? gT($sParentOption) : $sParentOption;
+                                        $sParentOption = ! empty($options[$sParentOption]) ? gT($options[$sParentOption]) : '';
```

Comment:
Can we use `??` instead please?

Reply:
?? is only for null, here it's for `'' `

Reply:
No, the check is on `empty`

Reply:
`$options[$sParentOption] ?? ''`

Reply:
> `$options[$sParentOption] ?? ''` should mean the same.

No , if `$options[$sParentOption] === ''` then you return `gT('')` we already have it in 5.X for help when help was not set by plugin (then are '' (and not null).

Reply:
Oh. So

```php
gT($options[$sParentOption] ?? '')
```

?

Reply:
Ah not exactly the same I guess.

Reply:
> No, the check is on `empty`

Empty returns false if string is empty.
That's why it is not the same.